frontend/form-builder/src/components/FormBoard/FormBoard.module.css

```
@@ -11,3 +11,16 @@
   font-size: 3rem;
   margin-bottom: var(--fds-spacing-4);
 }
+
+.droppedItem {
+  display: flex;
+  gap: var(--fds-spacing-4);
+  margin-bottom: var(--fds-spacing-4);
+  align-items: center;
+  font-size: 1.3rem;
+}
+
+.removalItem:hover {
+  cursor: pointer;
+  color: var(--fds-color-danger);
```

I don't think that variable exists in Designsystemet. Try this:
```suggestion
  color: var(--fds-semantic-surface-danger-hover);
```